.travis.yml

```diff
@@ -14,6 +14,12 @@ install: travis_retry composer install --prefer-source
 
 script: composer ci
 
+after_success:
+  - if [[ "`phpenv version-name`" != "7.1" ]]; then exit 0; fi
```

Comment:
Cool, did not think of using an exit :)

I suspect this version check does not work or is at least not very robust, if this indeed gives you the full PHP version including "beta" and whatnot. You are able to get the version specified in the yaml file here somehow tho.